ci-operator/config/kubevirt/kubevirt-ssp-operator/kubevirt-kubevirt-ssp-operator-master.yaml

```diff
@@ -0,0 +1,35 @@
+base_images:
+  base:
+    name: "4.6"
+    namespace: ocp
+    tag: base
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.13
+images:
+- dockerfile_path: build/Dockerfile
+  to: kubevirt-ssp-operator
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tag_specification:
+  name: "4.6"
+  namespace: ocp
+tests:
+- as: e2e-functests
+  commands: echo ${KUBEVIRT_SSP_OPERATOR}
+  dependencies:
```

Comment:
What are you trying to do with this? I don't see it in the [CONFIGURATION](https://github.com/openshift/ci-tools/blob/master/CONFIGURATION.md) manual? (Might be outdated though)

Reply:
I was trying to use this: https://steps.ci.openshift.org/help/ci-operator#literal-references
But it doesn't seem to work, `make ci-operator-config` removes it because of a marshalling error

Reply:
My knowledge might be outdated, but this might occur because of using the template strategy here.  In line 30 you are declaring to use the openshift installer src template which _might_ prohibit the steps approach (which was introduced a couple months ago AFAIR).

I understand steps vs. template to be two different strategies. The DPTP team has since told us that we should move towards the new steps strategy but we didn't get to it yet. AFAIU there is a different way to choose the cluster type in steps vs. using template, but that I don't know.

To be sure you might ask in #forum-testplatform.